website/docs/configuration/outputs.html.md

```diff
@@ -41,20 +41,27 @@ output "instance_ip_addr" {
 }
 ```
 
-The label immediately after the `output` keyword is the name that can be used
-to access this output in the parent module, if any, or the name that will be
-displayed to the user for output values in the root module.
+The label immediately after the `output` keyword is the name, which must be a
+valid [identifier](./syntax.html#identifiers). In a root module, this name is
+displayed to the user; in a child module, it can be used to access the output's
+value.
 
 The `value` argument takes an [expression](./expressions.html)
 whose result is to be returned to the user. In this example, the expression
 refers to the `private_ip` attribute exposed by an `aws_instance` resource
 defined elsewhere in this module (not shown). Any valid expression is allowed
 as an output value.
 
-Several other optional arguments are allowed within `output` blocks. These
-will be described in the following sections.
+## Accessing Child Module Outputs
 
-## Output Value Documentation
+In a parent module, outputs of child modules are available in expressions as
+`module.<MODULE NAME>.<OUTPUT NAME>`.
```

Comment:
Perhaps it would be helpful to also include a real example here, to imply that the pointy brackets are part of the placeholders and not literal. For example, we could lean on the example above and show something like `module.web_server.instance_ip_addr` as a concrete example.

Reply:
👍🏼